src/main/java/javaslang/collection/Map.java

```diff
@@ -224,10 +229,10 @@ default boolean contains(Tuple2<K, V> element) {
         Objects.requireNonNull(unzipper, "unzipper is null");
         return iterator().unzip(unzipper).map(Stream::ofAll, Stream::ofAll);
     }
-    
+
     @Override
     default <T1, T2, T3> Tuple3<Seq<T1>, Seq<T2>, Seq<T3>> unzip3(
-    		Function<? super Tuple2<K,V>, Tuple3<? extends T1, ? extends T2, ? extends T3>> unzipper) {
```

Comment:
Yes, do you have a hint how to do it?

I've mixed feelings with these tools because sometimes they seem to know more than the developer or produce false errors. However, these tools are great finding critical bugs. Using them should create value for us.